rsp/00x_bz_rest_api.md

```diff
@@ -0,0 +1,159 @@
+# RSP - 1 — *bz* REST API
+
+## `POST /login`
+
+### Synopsis
+
+Use this method to authenticate.
+
+### Request body
+
+```json
+{
+    "email": "$(email: String)",
+    "passwd": "$(passwd: String)"
+}
+```
+
+### Response body
+
+```json
+{
+    "id": "$(id: Int)",
```

Comment:
Linked to the discussion we are having on #11 this field may change in another format (maybe UUID)